include/envoy/stream_info/stream_info.h

```diff
@@ -128,6 +128,30 @@ struct ResponseCodeDetailValues {
   // despite the subsequent failure.
   const std::string LateUpstreamReset = "upstream_reset_after_response_started";
 
+  // The health check filter returned healthy to a health check.
```

Comment:
See my other comment about potentially not putting extension strings in the core code? I think they should live with the extensions? If you agree can you potentially clean up the others in this PR?

Reply:
I think my preference is to keep these strings with the extensions, because IMO it's not scalable with all the other custom extensions people might compile in/out. My real preference is to actually document all of these strings in the public docs. IMO once we allow adding this to access logs, we have to do this anyway. WDYT?

Reply:
I think having them in one place is far better for folks who don't know where to start looking, is consistent with the extension response flags all being in the same place, and is unlikely to have scaling problems any time soon.  I can agree to disagree though :-)

Reply:
If you feel strongly about it no problem. We can always revisit later. At this point I'm mostly starting to think about minimal size issues when compiling as a library, and also just cognitive scalabilty of an increasing number of extensions.

Reply:
Oh sorry I see that you already moved them. Do you want to revert and go back to what you had?

Reply:
Nah, we can always make some registry in the sky (which is friendly to non-upstream extensions) when/if debug becomes a problem.  And far be it from me to make Envoy-client-side harder than it will be :-)